javalin/src/main/java/io/javalin/http/Context.kt

```diff
@@ -39,13 +40,14 @@ open class Context(@JvmField val req: HttpServletRequest, @JvmField val res: Htt
     @get:JvmSynthetic @set:JvmSynthetic internal var endpointHandlerPath = ""
     @get:JvmSynthetic @set:JvmSynthetic internal var pathParamMap = mapOf<String, String>()
     @get:JvmSynthetic @set:JvmSynthetic internal var handlerType = HandlerType.BEFORE
+    @get:JvmSynthetic @set:JvmSynthetic internal var async = AtomicReference<CompletableFuture<*>?>(null)
```

Comment:
I think `async` should probably be called `atomicCompleteableFuture`, or at least `atomicFuture`. Further down where it says `async.get()`, I mistakenly thought you were calling `CompletableFuture#get`.